test/EFCore.PG.Tests/NpgsqlValueGeneratorSelectorTest.cs

```diff
@@ -210,4 +210,19 @@ public override int Next(EntityEntry entry)
         public override bool GeneratesTemporaryValues
             => false;
     }
+
+    [Fact]
+    public void CustomUuid7Test()
```

Comment:
**Nitpick:** Based on the naming of other tests here, I'd suggest `NpgsqlUuid7ValueGenerator_creates_uuidv7()`. In fact, that test probably belongs in a new test class specific to that class, in which case the name can just be `Creates_uuidv7()`.

Reply:
Good suggestion for name. Might want to just keep it in here. Reason being it is only temporary. Do we really want to create a new class, just for something temporary?

Reply:
Agreed, let's keep it here.

Reply:
Hm... Isn't the new generator here to stay? And only its implementation is going to be much simplified? It ought to have a test that confirms that it does what it claims, even when what it does is piggyback on .NET 9, right?

Reply:
Wouldn't that be doing the same thing as the other test mentioned further down then?